README.md

```diff
@@ -15,11 +15,11 @@ It automatically detects charts changed against the target branch.
 It is recommended to use the provided Docker image which can be [found on Quay](https://quay.io/helmpack/chart-testing/).
 It comes with all necessary tools installed.
 
-* Helm (http://helm.sh)
-* yamllint (https://github.com/adrienverge/yamllint)
-* yamale (https://github.com/23andMe/Yamale)
-* kubectl (https://kubernetes.io/docs/reference/kubectl/overview/)
-* Tooling for your cluster
```

Comment:
Wasn't sure what this meant exactly. Happy to add this back if we can get a little more specific.

Reply:
I think this can be removed.